docs/pod-metrics.md

```diff
@@ -40,3 +40,4 @@
 | kube_pod_spec_volumes_persistentvolumeclaims_info | Gauge | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `volume`=&lt;volume-name&gt;  <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-claimname&gt; | STABLE |
 | kube_pod_spec_volumes_persistentvolumeclaims_readonly | Gauge | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt;  <br> `volume`=&lt;volume-name&gt;  <br> `persistentvolumeclaim`=&lt;persistentvolumeclaim-claimname&gt; | STABLE |
 | kube_pod_status_scheduled_time | Gauge | `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; | STABLE |
+| kube_pod_status_unscheduled_reason | Gauge | `container`=&lt;container-name&gt; <br> `pod`=&lt;pod-name&gt; <br> `namespace`=&lt;pod-namespace&gt; <br> `reason`=&lt;Unschedulable&gt; <br> `message`=&lt;0/3 nodes are available: 3 Insufficient cpu&gt; <br> | STABLE |
```

Comment:
Since the metrics is named by `kube_pod_status_unscheduled_reason`, the `reason` label is redundant. BTW, `message` label can be renamed to `reason` label.

Reply:
Seems this has nothing to do with `container`.

Reply:
Good catch, in this case, I think the reason is specific, it's `Unschedulable`.So we can call this `kube_pod_status_unscheduled_message`? Or keep it with `kube_pod_status_unscheduled_reason`?

Reply:
We should not be storing the message in metrics . Just the reason should suffice.